assemblyai-starter/README.md

```diff
@@ -0,0 +1,47 @@
+# AssemblyAI Starter
+
+This is an Encore starter for a Conversational Intelligence app that uses [AssemblyAI](https://assemblyai.com). It lets you upload and transcribe voice data.
+
+## Developing locally
+
+When you have [installed Encore](https://encore.dev/docs/install), you can create a new Encore application and clone this example with this command.
+
+```bash
+encore app create my-app-name --example=conversational-intelligence
```

Comment:
This needs to be:
```
encore app create my-app-name --example=assemblyai-starter
```
because the folder name of the example is `assemblyai-starter`